app/src/main/java/com/example/nostack/ui/AttendeeHome.java

```diff
@@ -56,9 +76,25 @@ public void onCreate(Bundle savedInstanceState) {
     }
 
     @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container,
-                             Bundle savedInstanceState) {
-        // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_attendee_home, container, false);
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        // Inflate the layout for this fragment only once
+        View rootView = inflater.inflate(R.layout.fragment_attendee_home, container, false);
+        TextView userWelcome = (TextView) rootView.findViewById(R.id.text_userWelcome);
+
+        UserViewModel userViewModel = new ViewModelProvider((AppCompatActivity) getActivity() ).get(UserViewModel.class);
+        Log.d("AttendeeHome", "UserViewModel: " + userViewModel.getUser().getValue());
+        userViewModel.getUser().observe(getViewLifecycleOwner(), user -> {
+
+            if (user != null) {
+                Log.d("AttendeeHome", "Welcome, " + user.getFirstName() + "!");
+                userWelcome.setText("Welcome, " + user.getFirstName() + "!");
+            }
+            else{
+                Log.d("AttendeeHome", "User is null");
+            }
+        });
```

Comment:
example of state management of retrieving user model and setting the fragment textview to update to user first name